packages/core/src/Stores/ui-store.js

```diff
@@ -252,7 +261,16 @@ export default class UIStore extends BaseStore {
             should_show_cancel: observable,
             app_contents_scroll_ref: observable,
             is_deriv_account_needed_modal_visible: observable,
+            should_show_trade_assessment_form: observable,
+            should_show_appropriateness_warning_modal: observable,
+            should_show_warning_modal: observable,
```

Comment:
can we use more specific name for this observable? We have many warning modals and this name is little confusing.